CoffeeChat/View/OnboardingSearchDropdownView.swift

```diff
@@ -0,0 +1,153 @@
+    private let tableView = OnboardingSelectTableView()
+
+    // MARK: - Private Data Vars
+    private weak var delegate: OnboardingSearchViewDelegate?
+    private var placeholder: String!
+    private let reuseIdentifier = "OnboardingDropdownCell"
+    private var resultsTableData: [String] = []
+    private var tableData: [String]!
+
+    // MARK: - Private Constants
+    private let fieldsCornerRadius: CGFloat = 8
+
+    init(delegate: OnboardingSearchViewDelegate, placeholder: String, tableData: [String]) {
+        super.init(frame: .zero)
+        self.delegate = delegate
+        self.placeholder = placeholder
+        self.tableData = tableData
+        addViews()
+        setupConstraints()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    func addViews() {
+        searchBar.delegate = self
+        searchBar.backgroundColor = .backgroundWhite
+        searchBar.backgroundImage = UIImage()
+        searchBar.placeholder = placeholder
+        searchBar.setImage(UIImage(), for: .search, state: .normal) // Remove search icon from search bar.
+        if let textField = searchBar.value(forKey: "searchField") as? UITextField {
+            textField.backgroundColor = .backgroundWhite
+            textField.textColor = .black
+            textField.font = ._20CircularStdBook
+            textField.clearButtonMode = .never
+        }
+        searchBar.layer.cornerRadius = fieldsCornerRadius
+        searchBar.layer.shadowColor = UIColor.black.cgColor
+        searchBar.layer.shadowOffset = CGSize(width: 0.0 , height: 2.0)
+        searchBar.layer.shadowOpacity = 0.15
+        searchBar.layer.shadowRadius = 2
+        searchBar.searchTextPositionAdjustment = UIOffset(horizontal: -6, vertical: 0)
+        searchBar.showsCancelButton = false
+        addSubview(searchBar)
+
+        tableView.delegate = self
+        tableView.dataSource = self
+        tableView.isHidden = true // Initially hide the field data tableview.
+        tableView.allowsSelection = true
+        tableView.isScrollEnabled = true
+        tableView.separatorStyle = .none
+        tableView.isUserInteractionEnabled = true
+        tableView.backgroundColor = .darkGray
+        tableView.layer.cornerRadius = fieldsCornerRadius
+        tableView.bounces = false
+        tableView.register(OnboardingDropdownTableViewCell.self, forCellReuseIdentifier: reuseIdentifier)
+        addSubview(tableView)
+    }
+
+    func setupConstraints() {
+        searchBar.snp.makeConstraints { make in
+            make.leading.trailing.top.equalToSuperview()
+            make.height.equalTo(49)
+        }
+
+        tableView.snp.makeConstraints { make in
+            make.top.equalTo(searchBar.snp.bottom)
+            make.leading.trailing.equalToSuperview().inset(-10)
+        }
+    }
+
+    /// Hide search results table view, intended to be called by parent view controller.
+    func collapseTableView() {
```

Comment:
maybe rename to `hideTableView` since `collapseTableView` seems like we're doing something w/ animations